Demo/AuthenticatorDemoUITests/PrologueScreenObject.swift

```diff
@@ -0,0 +1,37 @@
+import ScreenObject
+import XCTest
+
+/// Slimmed down version of the one found in the WordPress iOS codebased.
+///
+/// Eventually, this should end up in a UI testing helper framework that the package exports, so that consumers can use it to write their
+/// own UI tests if they see fit.
```

Comment:
I hope to get onto this sooner rather than later.

Because of the end of year vacation period coming up, "sooner" will likely end up being Jan-Feb 2023.